include/boost/json/string.hpp

```diff
@@ -1134,6 +1134,26 @@ class string
         return {data(), size()};
     }
 
+#if ! defined(BOOST_JSON_STANDALONE) && \
+    ! defined(BOOST_NO_CXX17_HDR_STRING_VIEW)
+    /** Convert to a `std::string_view` referring to the string.
+
+        Returns a string view to the underlying character string. The size of
+        the view does not include the null terminator.
+
+        This overload is not defined when either `BOOST_JSON_STANDALONE` or
+        `BOOST_NO_CXX17_HDR_STRING_VIEW` is defined.
```

Comment:
@vinniefalco What do you think about this paragraph?

Reply:
its fine